### PR TITLE
Merge from eysispeisi/tboplayer

### PR DESCRIPTION
**Pull request by eysispeisi**
Improvements to windowed mode and added alsa support to options.

### CODE REVIEW

File: tboplayer.py [hunk 1]
```diff
@@ -2011,6 +2011,8 @@ def body(self, master):
         rb_local.grid(row=2,column=0,sticky=W)
         rb_auto=Radiobutton(master, text="Auto", variable=self.audio_var,value="auto")
         rb_auto.grid(row=3,column=0,sticky=W)
+        rb_auto=Radiobutton(master, text="ALSA", variable=self.audio_var,value="alsa")
```
Comment (heniotierra) [inline tboplayer.py]: I think this should not be a radio button for the same field, it should be a CHECKBOX for another field, and thus, you should also make sure that the resulting string for the audio option in the  command line follows the pattern  ```-o local:alsa```, and add a new this field to in the config file, making sure the setting can me remembered.

Comment (eysispeisi) [inline tboplayer.py]: In my case i have a usb sound card for my wireless headphones and i also use hdmi for my TV, I switch between those in the system menu. 
omxplayer option "-o alsa" uses what ever device is selected in the system menu (alsa).
I find it much more intuative to select my audio device in the system sound control than doing it in the player options.
that was the reasoning behind those changes anyways.

**Comment by eysispeisi** [comment]
Been doing a bit of digging on this, the problem i run into is there can be multiple audio devices in ALSA, f.e. my "Ear Force Stealth 400 CHAT" which i select using the system menu.
if the alsa options in tboplayer are to be mapped to specific devices, it would have to pull what devices are in ALSA in order to display all sound devices in tboplayer. That seems a bit overkill to me when we can simply keep it like it is with the alsa option simply selects the default device. which can be changed easily in system menues.
so in short i would like to keep the changes as is.

**Comment by eysispeisi** [comment]
Seeing kent's comment on this alsa issue, he's saying popcornmix is working on a solution, so maybe just omit the alsa change and only port the resizing change perhaps?

**Comment by heniotierra** [comment]
I see your point. Well, I guess we could merge your PR, since the way you did will most probably be supported after the changes on omxplayer. If it is not then it could be fixed later on. About the possibility of having multiple alsa devices, I agree it's too much effort, and for these cases, there's the possibility to use custom options.
